src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -44,6 +44,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- Resolve the TFM-specific attributes conditionally. -->
   <Choose>
+      <When Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '8.0')) ">
+        <PropertyGroup>
+          <_TargetingNETCoreApp30OrLater>true</_TargetingNETCoreApp30OrLater>
+          <_TargetingNET50OrLater>true</_TargetingNET50OrLater>
+          <_TargetingNET60OrLater>true</_TargetingNET60OrLater>
+          <_TargetingNET70OrLater>true</_TargetingNET70OrLater>
+          <_TargetingNET80OrLater>true</_TargetingNET80OrLater>
```

Comment:
What are all these lines intended to do? There is no such equivalent for roslyn so I don't know what this solves for razor.

Reply:
The 5.0 is used here:

https://github.com/dotnet/sdk/blob/c5bbadb84904b9e6f66b34247680893eaba4dc10/src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.Compilation.targets#L337-L353

The 6.0 is *seemingly* used here:

https://github.com/dotnet/sdk/blob/c5bbadb84904b9e6f66b34247680893eaba4dc10/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.5_0.targets#L439

But it's actually redefined in the same file, so it probably doesn't come from here.

Didn't find any usages of the others.

Reply:
Normally we used to define these variables every time we added a new version, since there were features keyed out of them.